EventFilter/CTPPSRawToDigi/interface/TotemSampicFrame.h

```diff
@@ -0,0 +1,263 @@
+/****************************************************************************
+*
+* This is a part of the TOTEM offline software.
+* Authors:
+*   Nicola Minafra
+*
+****************************************************************************/
+
+#ifndef EventFilter_CTPPSRawToDigi_TotemSampicFrame
+#define EventFilter_CTPPSRawToDigi_TotemSampicFrame
+
+#include <vector>
+#include <cstddef>
+#include <cstdint>
+#include <iomanip>
+#include <bitset>
+
+#include "EventFilter/CTPPSRawToDigi/interface/VFATFrame.h" 
+
+enum TotemSampicConstant {
+  NumberOfSamples = 24,
+};
+
+#pragma pack(push,1)
+struct TotemSampicData{
+  uint8_t      samples[TotemSampicConstant::NumberOfSamples];
+   
+  TotemSampicData() {}
+};
+#pragma pack(pop)
+
+#pragma pack(push,1)
+struct TotemSampicInfo{
+  uint8_t       hwId;
+  uint8_t       controlBits[6];
+  uint8_t       FPGATime[5];
+  uint16_t      TimeStampA;
+  uint16_t      TimeStampB;
+  uint16_t      CellInfo;
+  uint8_t       PlaneChannelId;
+  uint8_t       reserved[5];
+   
+  TotemSampicInfo() {}
+};
+#pragma pack(pop)
+
+#pragma pack(push,1)
+struct TotemSampicEventInfo{
+  uint8_t       hwId;
+  uint8_t       L1ATimeStamp[5];
+  uint16_t      bunchNumber;
+  uint32_t      orbitNumber;
+  uint32_t      eventNumber;
+  uint16_t      channelMap;
+  uint16_t      L1ALatency;
+  uint8_t       numberOfSamples;
+  uint8_t       offsetOfSamples;
+  uint8_t       FWVersion;
+  uint8_t       PLLInfo;
+   
+  TotemSampicEventInfo() {}
+};
+#pragma pack(pop)
+
+uint8_t GrayToBinary_8bit(const uint8_t &gcode_data)
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.
(Several examples in this code)